docs/sources/auth/enhanced_ldap.md

```diff
@@ -41,3 +41,25 @@ a user as member of a team and it will not be removed when the user signs in. Th
 5. Click on `Add group` button to save.
 
 <div class="clearfix"></div>
+
+## Active LDAP Synchronization
+
+In the open source version of Grafana, user data from LDAP will be synchronized only during the login process when authenticating using LDAP.
+
+With this feature you can configure Grafana to actively sync Grafana users with LDAP server(s) in the background. After certain configurable period it will map the groups with the already existing Grafana users, disable removed LDAP users, remove their auth tokens, etc.
```

Comment:
```suggestion
With this feature you can configure Grafana to actively sync users with LDAP server(s) in the background. Role and team membership will be updated, removed users will be deleted and their auth tokens removed. Only users that have logged into Grafana at least once will be synchronized.```

Reply:
The sentence felt a bit weird to me so I rewrote it a bit, what do you think?

Reply:
Sounds nice! 

Only 

>  will be deleted

You meant disabled, right?

Reply:
Right, good catch!